tests/xtimer_cli/tests/01__xtimer_base.robot

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
+Documentation       Basic tests to verify functionality of the Xtimer API.
+
+Suite Setup         Run Keywords    Reset Application
+...                                 DUT Must Have API Firmware
+Test Setup          Run Keywords    Reset Application
+...                                 DUT Must Have API Firmware
```

Comment:
As this is somehow a POC/showcase for using robot framework a little more comments here would be nice to make clear what `Suite Setup`, `Test Setup` and so on actually does. 
Also IIRC the second `DUT Must Have API Firmware` is a little hack to ensure the communication channel is working correctly. -> this should be noted explicitly to make it clear to others that this is required.

Reply:
first: why should I repeat the RF documentation? There is plenty documentation on Suite/Test Setup at the original sources, I can add some links in the dist/robotframework/README.md but copy-pasting docu doesn't make sense (to me).

second: no that's no hack in that sense, it's just there to ensure that before each test the devices are reset and responding correctly before running the tests. The Suite Setup could be avoided but on the other it helps to skip all tests if there is a problem with the board.